src/main/java/mate/academy/rickandmorty/controller/RickAndMortyController.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Character management", description = "Endpoints for managing characters")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/character")
```

Comment:
Better to use plural as endpoint represents that we have collection of characters
```suggestion
@RequestMapping("/characters")
```